pulp.spec

```diff
@@ -409,6 +409,7 @@ Pulp provides replication, access, and accounting for software repositories.
 %{_bindir}/pulp-manage-db
 %{_bindir}/pulp-qpid-ssl-cfg
 %{_bindir}/pulp-gen-ca-certificate
+%{_bindir}/pulp_streamer
```

Comment:
Does the spec file actually need this entry? What are the files contents? I figured the `entry_point` would populate the file when `setup.py` is run.